src/GafferSceneUI/Inspector.cpp

```diff
@@ -454,7 +454,12 @@ Gaffer::EditScope *Inspector::targetEditScope() const
 		return nullptr;
 	}
 
-	return runTimeCast<EditScope>( m_editScope->getInput()->node() );
+	return PlugAlgo::findSource(
```

Comment:
Nice use of `findSource()`! I always forget we've got some of these nice little utilities.

Reply:
Yeah! I was very pleased to find it so cleanly usable here.